osbenchmark/utils/git.py

```diff
@@ -121,7 +121,7 @@ def branches(src_dir, remote=True):
     if remote:
         # alternatively: git for-each-ref refs/remotes/ --format='%(refname:short)'
```

Comment:
May want to update this comment, and mention the difference in the behavior of git versions.